releasenotes/notes/cleanup-timeline-drawer-a6287bdab4459e6e.yaml

```
@@ -0,0 +1,27 @@
+---
+upgrade:
```

Should this be a new feature note?
```suggestion
feature:
```